src/utils/RegHubUtil.js

```diff
@@ -37,7 +38,7 @@ module.exports = {
 
     searchReq = request.get({
       url: 'https://registry.hub.docker.com/v1/search?',
-      qs: {q: query, page}
+      qs: {q: query, page: page, time: +new Date()}
```

Comment:
`+new Date()` creates a date object that is then coerced into a number (same as `Date.now()`)  - I only have it to make sure cached response isn't seen, but could be removed, or `Date.now()` can be used in its place.